browser/src/slideshow/PresenterConsole.js

```diff
@@ -561,6 +561,11 @@ class PresenterConsole {
 
 		mainContentContainer.append(this.tooltip);
 
+		this._tooltip = L.control.tooltip({
```

Comment:
why do we create new tooltip here while we set the style above?
Shouldn't that be mooved above styling?

Reply:
I have setup an initial code to simplify tooltip, @Darshan-upadhyay1110 should modify according to his needs =)

Reply:
ah so it's todo, ok